tests/acceptance/acceptance-test.js

```diff
@@ -1,88 +1,67 @@
 import {
-  afterTooltipRenderChange,
   assertTooltipNotRendered,
   assertTooltipRendered,
   assertTooltipNotVisible,
+  findTooltip,
   triggerTooltipTargetEvent,
   assertTooltipVisible,
-} from '../../tests/helpers/ember-tooltips';
+} from 'ember-tooltips/test-support';
 
-/* globals andThen, visit */
-moduleForAcceptance('Acceptance | acceptance');
+module('Acceptance | acceptance', function(hooks) {
+  setupApplicationTest(hooks);
 
-test('all acceptance tests', function(assert) {
+  test('all acceptance tests', async function(assert) {
+    assert.expect(11);
 
-  assert.expect(12);
+    await visit('/acceptance');
 
-  visit('/acceptance');
-
-  const tooltipOrPopoverSelector = '.ember-tooltip, .ember-popover';
-
-  andThen(() => {
-    assert.equal($(tooltipOrPopoverSelector).length, 0,
-        'initially there should be 0 tooltips or popovers rendered');
-  });
-
-  /* Begin tooltip tests */
-
-  andThen(() => {
+    assertTooltipNotRendered(assert);
 
     const $tooltipTarget = $('.js-test-tooltip-target');
-    const options = {
+    const tooltipOptions = {
       selector: '.js-test-tooltip',
     };
 
     assert.equal($tooltipTarget.length, 1, 'there should be one $tooltipTarget');
 
-    assertTooltipNotRendered(assert, options);
+    assertTooltipNotRendered(assert, tooltipOptions);
 
-    triggerTooltipTargetEvent($tooltipTarget, 'mouseenter');
+    await triggerTooltipTargetEvent($tooltipTarget, 'mouseenter');
 
-    assertTooltipRendered(assert, options);
+    assertTooltipRendered(assert, tooltipOptions);
 
-    assertTooltipVisible(assert, options);
+    assertTooltipVisible(assert, tooltipOptions);
 
-    triggerTooltipTargetEvent($tooltipTarget, 'mouseleave');
+    await triggerTooltipTargetEvent($tooltipTarget, 'mouseleave');
 
-    afterTooltipRenderChange(assert, () => {
-      assertTooltipNotVisible(assert, options);
-    }); // Default hideDelay = 250
+    await settled();
 
-  });
-
-  /* Begin popover tests */
-
-  andThen(() => {
+    assertTooltipNotVisible(assert, tooltipOptions);
 
     const $popoverTarget = $('.js-test-popover-target');
-    const options = {
+    const popoverOptions = {
       selector: '.js-test-popover',
     };
 
     assert.equal($popoverTarget.length, 1, 'there should be one $popoverTarget');
 
-    assertTooltipNotRendered(assert, options);
+    assertTooltipNotRendered(assert, popoverOptions);
 
-    triggerTooltipTargetEvent($popoverTarget, 'mouseenter');
+    await triggerTooltipTargetEvent($popoverTarget, 'mouseenter');
 
-    assertTooltipRendered(assert, options);
+    assertTooltipRendered(assert, popoverOptions);
 
-    assertTooltipVisible(assert, options);
+    assertTooltipVisible(assert, popoverOptions);
 
-    triggerTooltipTargetEvent($popoverTarget, 'mouseleave');
+    await triggerTooltipTargetEvent($popoverTarget, 'mouseleave');
 
-    afterTooltipRenderChange(assert, () => {
-      assertTooltipNotVisible(assert, options);
```

Comment:
should this assertion be preserved?

Reply:
Yes, this looks like an error. I will add the assertion back.